src/Controller/ListPageRssController.php

```diff
@@ -393,33 +405,32 @@ protected function getChannelDescription(NodeInterface $node, CacheableMetadata
       return '';
     }
 
-    $facet_storage = $this->entityTypeManager->getStorage('facets_facet');
-
+    $facets = $this->getKeyedFacetsFromSource($list_source);
```

Comment:
Here we should not load the facets from scratch but use the ones from the facets manager which were already filled with results. Even though we don't technically use these results, but always better to use the facets manager.